src/main/kotlin/org/wfanet/measurement/kingdom/deploy/gcloud/spanner/kingdom.sdl

```diff
@@ -195,57 +138,26 @@ CREATE TABLE Reports (
   State             INT64 NOT NULL,  -- Proto enum encoded as int
 
   -- ReportDetails serialized proto
-  ReportDetails     BYTES(MAX) NOT NULL,
-  ReportDetailsJson STRING(MAX) NOT NULL,
-) PRIMARY KEY (AdvertiserId, ReportConfigId, ScheduleId, ReportId),
-  INTERLEAVE IN PARENT ReportConfigSchedules ON DELETE CASCADE;
-
--- Enable idempotent Report creation from ReportConfigs.
-CREATE INDEX ReportsByTimeWindow
-  ON Reports(AdvertiserId, ReportConfigId, WindowStartTime, WindowEndTime),
-  INTERLEAVE IN ReportConfigs;
+  MeasurementDetails     BYTES(MAX) NOT NULL,
+  MeasurementsDetailsJson STRING(MAX) NOT NULL,
+) PRIMARY KEY (MeasurementConsumerId, MeasurementId);
 
--- Enable finding reports ready to be worked on.
-CREATE INDEX ReportsByState ON Reports(State, UpdateTime ASC);
+-- Enable finding Measurements ready to be worked on.
+CREATE INDEX MeasurementsByState ON Measurements(State, UpdateTime ASC);
 
-CREATE UNIQUE INDEX ReportsByExternalId
-  ON Reports(ExternalReportId, CreateTime DESC);
+-- Enable finding Measurements by externally generated Foreign ids
+CREATE INDEX MeasurementsByProvidedMeasurementId
+  ON Measurements(MeasurementConsumerId, ProvidedMeasurementId);
 
--- Contains status updates from the Duchies and within the Kingdom for a
--- particular report. For any given report, each Duchy might send many updates
--- (one or more per stage of the MPC protocol). This is used to give a bird's
--- eye view of the state of the computations to help debug and track progress.
-CREATE TABLE ReportLogEntries (
-  AdvertiserId    INT64 NOT NULL,
-  ReportConfigId  INT64 NOT NULL,
-  ScheduleId      INT64 NOT NULL,
-  ReportId        INT64 NOT NULL,
-  CreateTime      TIMESTAMP NOT NULL OPTIONS (allow_commit_timestamp = true),
-
-  -- ReportLogDetails serialized proto.
-  ReportLogDetails      BYTES(MAX) NOT NULL,
-  ReportLogDetailsJson  STRING(MAX) NOT NULL,
-) PRIMARY KEY (AdvertiserId, ReportConfigId, ScheduleId, ReportId, CreateTime),
-  INTERLEAVE IN PARENT Reports ON DELETE CASCADE;
-
-CREATE TABLE ReportConfigCampaigns (
-  AdvertiserId    INT64 NOT NULL,
-  ReportConfigId  INT64 NOT NULL,
-  DataProviderId  INT64 NOT NULL,
-  CampaignId      INT64 NOT NULL,
-
-  CONSTRAINT FK_Campaign
-    FOREIGN KEY (DataProviderId, CampaignId)
-    REFERENCES Campaigns(DataProviderId, CampaignId),
-) PRIMARY KEY (AdvertiserId, ReportConfigId, DataProviderId, CampaignId),
-  INTERLEAVE IN PARENT ReportConfigs ON DELETE CASCADE;
+CREATE UNIQUE INDEX MeasurementsByExternalId
+  ON Measurements(ExternalMeasurementId, CreateTime DESC);
 
 -- The Metric value is actually stored by the Duchy. The Duchy has a map from
```

Comment:
Replace "Metric value" with "Requisition data" or something else? We don't have "Metric value" any more.